src/ServiceBusExplorer/Forms/MainForm.cs

```diff
@@ -4041,7 +4041,14 @@ private void SetTitle(string prefix)
         #region Public Static Methods
         public static void StaticWriteToLog(string message, bool async = true)
         {
-            mainSingletonMainForm.WriteToLog(message);
+            if(null != mainSingletonMainForm)
```

Comment:
Always found this notation awkward to read.
If you’re concerned about accidental null assignment, there’s a more readable way.

```suggestion
            if(mainSingletonMainForm is not null)
```

Reply:
Will need to revert. Or bump C# language version.

Reply:
Sadly we are stuck on 7.3 as long as we are using .Net Framework. https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version.